src/metabase/integrations/slack.clj

```diff
@@ -300,6 +313,22 @@
         (log/error (u/format-color 'red message))
         (throw (ex-info message {:status-code 400}))))))
 
+(defn bug-report-channel
```

Comment:
i'm no clojurian, but maybe `set-bug-report-channel` would be more descriptive here?